app.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+
+const app = express()
+
+app.use(express.static('public'))
+
+app.get('/',(req,res,next) => {
+    res.redirect('/home')
+})
+
+app.get('/home',(req,res,next) => {
+    res.sendFile(__dirname + '/views/home.html')
+})
```

Comment:
Esta forma está bien, pero mejor que utilicéis esta otra, ya que con render puedes pasar parametros :

`app.get('/', (req, res, next) => res.render('home'))`

En este ejemplo está la home en '/' que es la ruta habitual de la home.